app/models/route.rb

```diff
@@ -0,0 +1,9 @@
+class Route < ApplicationRecord
+  belongs_to :user
+  has_many :waypoints
+  has_one :contact, through: :user
+
+# default status is 0, active
+# maybe we don't even need this if we are going to destroy upon checking in
```

Comment:
Definitely something to think about as we start integrating front end and back end together.